webapp2_extras/sessions_memcache.py

```diff
@@ -28,4 +28,3 @@
     'webapp2_extras.sessions_memcache is deprecated. '
     'App Engine-specific modules were moved to webapp2_extras.appengine.'),
     stacklevel=1)
-from webapp2_extras.appengine.sessions_memcache import *
```

Comment:
leave these for now, just [ignore](http://flake8.pycqa.org/en/latest/user/ignoring-errors.html#in-line-ignoring-errors) them.